1-exercises/B-setting-values/exercise2.js

```diff
@@ -25,7 +25,9 @@ let student = {
   - Use bracket notation to change the value of hasPassed
 */
 
-// write code here
+if (student["attendance"] >= 90 && student["examScore"] > 60) {
```

Comment:
You can use both `student.attendance` and `student["attendance"]` - former is probably easier to read.